# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/ClassifierDummyChannel.cpp
# Commit: b4b920f24bed
# Full Hash: b4b920f24bed3d02a7000892e9594c515970e1f6
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-03 09:24:59
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 3: Add mechanism to notify EarlyHintPreloader that HttpChannelParent is ready to complete the redirect r=necko-reviewers,kershaw,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161178
# ==============================================================================

diff -r 13a9f4631369 -r b4b920f24bed netwerk/protocol/http/ClassifierDummyChannel.cpp
--- a/netwerk/protocol/http/ClassifierDummyChannel.cpp	Fri Dec 02 16:15:53 2022 +0000
+++ b/netwerk/protocol/http/ClassifierDummyChannel.cpp	Fri Dec 02 16:15:54 2022 +0000
@@ -812,6 +812,18 @@
   return NS_ERROR_NOT_IMPLEMENTED;
 }
 
+NS_IMETHODIMP
+ClassifierDummyChannel::SetEarlyHintPreloaderId(
+    uint64_t aEarlyHintPreloaderId) {
+  return NS_ERROR_NOT_IMPLEMENTED;
+}
+
+NS_IMETHODIMP
+ClassifierDummyChannel::GetEarlyHintPreloaderId(
+    uint64_t* aEarlyHintPreloaderId) {
+  return NS_ERROR_NOT_IMPLEMENTED;
+}
+
 void ClassifierDummyChannel::SetConnectionInfo(
     mozilla::net::nsHttpConnectionInfo* aInfo) {}
 